.github/workflows/build.yaml

```diff
@@ -16,6 +16,9 @@ name: Builds
 
 on:
     push:
+        branches:
+            - master
+            - 'v*-branch'
         branches-ignore:
```

Comment:
do we need an ignore at all given that we have `branches` element?

Reply:
Removed the `branches-ignore` element from the push event.